docs/README.md

```diff
@@ -144,6 +144,10 @@ In addition to that, keep in mind the message variables for each block since you
 | `savingsWithTax` | `string` | Difference between previous product price and the new one with taxes. |
 | `savingsPercentage` | `string` | Percentage of savings. |
 
+| Prop name | Type | Description | Default value |
+| --- | --- | --- | --- |
+| `usingSpotPrice` | `boolean` | Calculate discounts based on the Spot Price | `false` |
```

Comment:
```suggestion
| `usingSpotPrice` | `boolean` | Whether the product spot price should be used for discount calculation (`true`) or not (`false`). | `false` |
```

Reply:
@guerreirobeatriz I've removed this from Doc, now the prop is on ListPrice component.